minio/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM mikesir87/aws-cli:latest
```

Comment:
maybe `Dockerfile.aws-cli` to make clear this is not the server's Dockerfile, but the client's one?

Reply:
done.